source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -145,10 +156,64 @@ corresponding code.
          :end-before: end-change-stream-pipeline
          :language: csharp
 
-To learn more about modifying your change stream output, see the
-:manual:`Modify Change Stream Output
-</changeStreams/#modify-change-stream-output>` section in the {+mdb-server+}
-manual.
+Split Large Change Events Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your application generates change events that exceed 16 MB in size, the
+server returns a ``BSONObjectTooLarge`` error. To avoid this error, you can use
+the ``$changeStreamSplitLargeEvent`` pipeline stage to split the events
+into smaller fragments.
+
+After you receive the change stream event fragments, you can use the
```

Comment:
I think it would be better to show how to use `ChangeStreamSplitLargeEvent` first (as done in the last code section) an then show how to merge fragments, as this is a secondary aspect.

Reply:
Since I use the helper methods in the ChangeStreamSplitLargeEvent example, I decided to introduce the helper methods first - but the actual split event code example is the focus of the section so it does make sense to include it first. Updated!